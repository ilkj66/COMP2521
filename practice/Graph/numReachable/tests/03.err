numReachable.c:24:44: runtime error: load of value 190, which is not a valid value for type 'bool'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior numReachable.c:24:44 in 
numReachable.c:14:13: runtime error: load of value 190, which is not a valid value for type 'bool'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior numReachable.c:14:13 in 
