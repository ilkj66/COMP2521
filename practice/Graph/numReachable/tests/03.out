Enter number of vertices: Enter number of edges: Enter 12 edges in the form `v w` (no quotes):
Number of vertices: 11
Number of edges: 12
Edges:
 0: 5
 1: 2 3 4
 2: 1 3 4
 3: 1 2 4
 4: 1 2 3
 5: 0 6
 6: 5 7
 7: 6 8
 8: 7 9
 9: 8 10
10: 9

Enter the source vertex: Number of vertices reachable from vertex 3: 4
Enter the source vertex: Number of vertices reachable from vertex 9: 7
Enter the source vertex: 